server/game/cards/03-FF/Phantasm.js

```diff
@@ -0,0 +1,93 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class Phantasm extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Phantasm: Move Unbooted Dude',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select unbooted dude to move',
+                cardCondition: { 
+                    location: 'play area', 
+                    controller: 'opponent', 
+                    condition: card => 
+                        card.gamelocation === this.gamelocation ||
+                        card.getGameLocation().isAdjacent(this.gamelocation) &&
+                        !card.booted
```

Comment:
This condition lets you choose also booted dudes at the caster's location that is not correct. That is because && has precedence over ||.
The condition should be:

```suggestion
                        (card.gamelocation === this.gamelocation ||
                        card.getGameLocation().isAdjacent(this.gamelocation)) &&
                        !card.booted
```